conf.py

```diff
@@ -183,4 +183,10 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {"https://docs.python.org/": None}
+intersphinx_mapping = {
```

Comment:
this is very useful ! Thanks

Reply:
Yeah, I just started looking into this recently.  It also allows easy referencing of labels or classes/functions etc. in the other documentation.